src/MessageHandler/ActivityPub/Inbox/ActivityHandler.php

```diff
@@ -113,6 +107,13 @@ private function handle(?array $payload)
                     }
                 }
             }
+        } elseif ('Create' === $payload['type']) {
+            $actor = $this->manager->findActorOrCreate($payload['actor']);
+            if ($actor instanceof Magazine && $actor->lastOriginUpdate < (new \DateTime())->modify('-3 hours')) {
+                $actor->lastOriginUpdate = new \DateTime();
+                $this->entityManager->persist($actor);
+                $this->entityManager->flush();
+            }
```

Comment:
What actually needs checking is the announced type, so payloads like this will trigger the code:
```json
{
  "type": "Announce",
  "object": {
    "type": "Create",
    ...
  },
  ...
}
```
So I'd say move it back to where it was before and change the condition to:
```php
if ('Announce' === $payload['type']) {
    if (\is_array($payload['object'])) {
        if ('Create' === $payload['object']['type']) {
```

Reply:
I would've thought that means it'd only update on comments that come in from external instances, but maybe I'm misunderstanding AP here. I assumed create = comes from owner instance, announce = comes from another instance, and owner is announcing it happened. Or maybe owner instances both create and announce create their own posts?

Reply:
Yeah a create from a magazine and an announced create from a magazine should refresh the field. Until now the plain create did not do that, but it should, you're right

Reply:
Ok, that might actually explain some things I had seen. Looking at this one magazine, it says "This magazine is not receiving updates (last activity 29 day(s) ago). " but has posts from today, yesterday etc, continuing to get posts, but they're all posts from that instance the magazine is from

So if I understand correctly, I need both sections, what I have here which updates from the origin's creates, and then what you suggested which updates when the origin announces creates as well

Reply:
Well you alsp just get the creates from the user directly which would not count towards origin uodates.

And yes both sections would be good indeed

Reply:
In this conversation I forgot the point I made in my latest comment. Sorry for the confusion on my part